hmvec/cosmology.py

```diff
@@ -4,6 +4,9 @@
 import camb
 from camb import model
 import scipy.interpolate as si
+import scipy.constants as constants
+from scipy.integrate import simps
+import tqdm
```

Comment:
Is this used anywhere?

Reply:
The question is about both tqdm and simps.

Reply:
Not anymore. I've removed them.